cspell.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
can we move this file to `cspell` folder?

Reply:
Done.